### PR TITLE
Fixes 602: Add SSL configuration

### PR DESCRIPTION
**Pull request by Andrewgdewar**

### CODE REVIEW

File: pkg/config/config.go [hunk 1]
```diff
@@ -260,7 +261,7 @@ func Load() {
 
 	if len(clowder.KafkaServers) > 0 {
 		log.Warn().Msgf("clowder.KafkaServers has length of %s", strconv.Itoa(len(clowder.KafkaServers)))
-		LoadedConfig.NotificationsClient = SetupNotifications(clowder.KafkaServers)
+		LoadedConfig.NotificationsClient = SetupNotifications(clowder.KafkaServers, LoadedConfig.Kafka.Capath)
```
Comment (jlsherrill) [inline pkg/config/config.go:264]: i don't think this is quite right, we don't load the clowder() kafka config into our own.  i think you need to pull it from clowder: 

``` ca, err := clowder.LoadedConfig.KafkaCa(clowder.LoadedConfig.Kafka.Brokers...)```

you'll need to wrap it in a clowder.IsClowderEnabled()  too

Comment (jlsherrill) [reply pkg/config/config.go:264]: actually, i'm wrong, but lets log the Capath while we're at it?

**Comment by rverdile** [comment]
should this be Refs or Fixes?

**Comment by Andrewgdewar** [comment]
> 

Well it's attempting to fix something broken? 😞

**Comment by jlsherrill** [comment]
yeah normally, you use Refs for initial commits before the pr that fixes it, or you may use Refs for a small followup pr, but this maybe is a little different, not entirely optimal situation (where we have to test in stage)

**Pull request merged**